File: problemas/iniciante/2061/2061.cpp
#include <iostream>

using namespace std;

int main()
{
    int N, M;
    string acao;

    cin >> N >> M;
    for (int i = 0; i < M; ++i)
    {
        cin >> acao;

        if (acao == "fechou")
        {
            ++N;
        }
        else
        {
            --N;
        }
    }

    cout << N << endl;

    return 0;
}

File: problemas/iniciante/2061/2061.r
input <- file('stdin', 'r')

entrada <- as.integer(strsplit(readLines(input, n = 1), ' ')[[1]])
n <- entrada[1]
m <- entrada[2]
for(i in 1:m) {
  acao <- readLines(input, n = 1)

  if(acao == 'fechou'){
    n <- n + 1
  } else {
    n <- n - 1
  }
}

write(paste(n), '')